WPMedia/fusion-news-theme-blocks
# THEMES-1916: Full Author was not accounting for "original" author data

This is a correction for an issue found in testing.  The full-author-bio-block was not correctly identifying images since it was using the additiona_properties.original author data which is not scanned by the signing service.

## Description

#### Jira Ticket: [THEMES-1916](https://arcpublishing.atlassian.net/browse/THEMES-1916)

## Test Steps

1. Checkout this branch `git checkout themes-1916-2`
2. Run fusion repo with linked blocks `npx fusion start -f -l`
3. Setup a page with content-api: `/2022/06/01/what-is-an-article/`and both author blocks.
![image](https://github.com/WPMedia/arc-themes-blocks/assets/2287238/5285c969-beac-4cac-a366-9396079841a1)
4. Verify both author images have resizer information.


[THEMES-1916]: https://arcpublishing.atlassian.net/browse/THEMES-1916?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
